x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -140,7 +140,7 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
   const { formatUrl } = useFormatUrl(SecurityPageName.rules);
   const [showBuildingBlockAlerts, setShowBuildingBlockAlerts] = useState(false);
   const [showOnlyThreatIndicatorAlerts, setShowOnlyThreatIndicatorAlerts] = useState(false);
-  const loading = userInfoLoading || listsConfigLoading || isLoadingIndexPattern;
+  const loading = userInfoLoading || listsConfigLoading;
```

Comment:
Same [problem](https://github.com/elastic/kibana/pull/126105/files#r819676130) with the `indexPatternsLoading` here. This was added by @marshallmain a few weeks ago to prevent _AlertsCountPanel_ and _AlertsHistogramPanel_ to render while loading the `runtimeMappings`, after talking with him the loader has been moved to those components individually.